Topic4/src/interfaces/User.java

```diff
@@ -0,0 +1,24 @@
+package interfaces;
+
+import java.awt.Image;
+
+public interface User {

```

Comment:
Because I wanted to model all interfaces, it was not a good idea apparently, I'll change it, sorry